crates/goose/src/prompt_template.rs

```diff
@@ -0,0 +1,147 @@
+// prompt.rs
+
+use serde::Serialize;
+use std::fs;
+use std::path::PathBuf;
+use tera::{Context, Error as TeraError, Tera};
+
+/// Get the path to the prompts directory
+fn prompts_dir() -> PathBuf {
+    // Option 1: Use env var for configuration
+    if let Ok(prompts_dir) = std::env::var("GOOSE_PROMPTS_DIR") {
```

Comment:
nit: when would you use an external prompts dir, is this like an overwrite by the user option?

Reply:
gonna remove this. had this in an older version but don't think we'll need it